blackbox/docs/admin/user-management.rst

```diff
@@ -46,7 +46,16 @@ To create a new user for the CrateDB database cluster use the
     cr> CREATE USER user_a;
     CREATE OK, 1 row affected (... sec)
 
-The newly created user does not have any special :ref:`administration-privileges`.
+.. TIP::
+
+    Newly created users do not have any privileges. After creating a user, you
+    should :ref:`configure user privileges <administration-privileges>`.
+
+    For example, to grant all privileges to the ``user_a`` user, run::
+
+        cr> GRANT ALL PRIVILEGES TO user_a;
+        GRANT OK, 3 rows affected (... sec)
```

Comment:
this privileges must be remove by a hidden statement so followup tests will succeed, see jenkins failure